layouts/index.html

```diff
@@ -80,28 +59,33 @@ <h2 class="project-title">TimeClock</h2>
       <div class="section companies">
         <h2>You may have seen me working at these companies...</h2>
         <div class="company-list">
-          <div class="company oomba">
-          </div>
-          <div class="company amazon">
-          </div>
-          <div class="company disney">
-          </div>
+          <a class="company oomba" href="resume#oomba-inc">
```

Comment:
The hypothetical data object mentioned above can also have information regarding the resume link name as well.